libs/blocks/global-navigation/global-navigation.js

```diff
@@ -466,17 +466,15 @@ class Gnav {
       // reset sign up value on change
       children[0].attributes.isSignUpRequired = false;
 
-      if (isDesktop.matches) {
```

Comment:
I remember there was a `change` event listener for `isDesktop`, L509 in this file. Is that still needed?

Reply:
@overmyheadandbody - I was thinking the same, but thought about the following flow: user has a device with width suitable for desktop, the universal nav is rendered; then user rotates the device and it goes into mobile width; without reinitialising the universal navigation I am concerned that some previous widths, positions will be preserved and the dropdowns might appear to be detached from the navigation. I think it's safer to reload the navigation when viewport changes.

Reply:
Good catch! Thanks 👍 We won't need to reload the unav on switching between desktop and mobile viewport anymore as it should always be visible. I made the change.